### PR TITLE
Updating BitLocker recovery API

### PR DESCRIPTION
**Pull request by hafowler**
New root is /informationProtection instead of /bitlocker

### CODE REVIEW

**Review by davidmu1 — APPROVED** [review]

File: concepts/changelog.md [hunk 1]
```diff
@@ -1967,6 +1967,7 @@ Microsoft Search is now exposing a way to search and index data in Microsoft Gra
 | Addition        | Beta        | Added **synchronizationJobSettings** property to the [synchronizationJob](/graph/api/resources/synchronization-synchronizationjob?view=graph-rest-beta&preserve-view=true) resource.        |
 | Addition        | Beta        | Added new possible values for the **reason** property in the [synchronizationQuarantine](/graph/api/resources/synchronization-quarantine?view=graph-rest-beta&preserve-view=true) resource. |
 | Addition        | Beta        | Added the capability to manage password single sign on credentials on the [servicePrincipal object](/graph/api/resources/serviceprincipal?view=graph-rest-beta&preserve-view=true).         |
+| Change          | Beta        | Updated [BitLocker recovery](/graph/api/resources/bitlockerrecoverykey?view=graph-rest-beta&preserve-view=true) endpoint to fall under /informationProtection. |
```
Comment (Lauragra) [inline concepts/changelog.md:1970]: This change is listed under September 2019. Please move to December 2020. Please also include an update to the changelog JSON file.

Comment (hafowler) [reply concepts/changelog.md]: @Lauragra I updated the changelog.md file. 
For the changelog JSON file - I originally had the Microsoft.BitLocker.json file - what in that to I need to change? Where do I need to put "informationProtection"?

Comment (Lauragra) [reply concepts/changelog.md]: @hafowler , I'm not sure what's happening, but your PR currently includes over 1,000 files. Can you please look into that? You can create a new JSON file named Microsoft.InformationProtection.json and add your changelist entry in there.

Comment (hafowler) [reply concepts/changelog.md]: I am not sure what happened here either. I am going to close this PR, and create a new one, as that would be the simplest move forward for me.

**Comment by hafowler** [comment]
Closing this PR, as something weird happened when I tried to pull the directory. Will open a new PR to include these changes

**Pull request closed without merging**